src/test/e2e/services/session-service.e2e.test.ts

```diff
@@ -114,6 +114,7 @@ test('Can delete session by sid', async () => {
 
 test('Can delete stale sessions', async () => {
     await sessionService.insertSession(newSession);
+    await new Promise((resolve) => setTimeout(resolve, 100)); // Ensure a different createdAt
```

Comment:
How about something in the middle?


```suggestion
    await new Promise((resolve) => setTimeout(resolve, 50)); // Ensure a different createdAt
```

```suggestion
    await new Promise((resolve) => setTimeout(resolve, 10)); // Ensure a different createdAt
```